compiler/src/dmd/common/file.d

```diff
@@ -129,7 +129,8 @@ struct FileMapping(Datum)
                 enum openFlags = CREATE_ALWAYS;
             }
 
-            handle = filename.asDString.extendedPathThen!(p => CreateFileW(p.ptr, createFileMode, 0, null, openFlags, FILE_ATTRIBUTE_NORMAL, null));
+            handle = filename[0 .. strlen(filename)].
```

Comment:
if this duplicates `root.string.toDstring()` then why are you using `filename[0 .. strlen(filename)]` instead of `filename.toDstring`?

Reply:
1. nothing else in `common` imports from `root`
2. `toDstring` is trivia as well
3. the less mutual entanglement the various modules have, the better

Reply:
Fair enough